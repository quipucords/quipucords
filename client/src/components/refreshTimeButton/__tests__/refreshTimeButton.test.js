import React from 'react';
import { mount } from 'enzyme';
import RefreshTimeButton from '../refreshTimeButton';

describe('RefreshTimeButton Component', function() {
  it('should render', () => {
    const props = {
      onRefresh: jest.fn()
    };

    const component = mount(<RefreshTimeButton {...props} />);

    expect(component.render()).toMatchSnapshot();
  });
});
