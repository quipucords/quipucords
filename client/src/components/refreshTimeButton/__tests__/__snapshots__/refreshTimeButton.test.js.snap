// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RefreshTimeButton Component should render 1`] = `
<button
  class="refresh-button btn btn-link"
  type="button"
>
  <span
    aria-hidden="true"
    class="fa fa-refresh"
  />
  <span
    class="last-refresh-time"
  >
    Refreshed 
  </span>
</button>
`;
